.github/workflows/integrate.yaml

```diff
@@ -57,11 +57,13 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
       - name: Setup operator environment
-        uses: charmed-kubernetes/actions-operator@main
+        uses: claudiubelu/actions-operator@18ebf92ae3043bd3dd15238e5d9b662d7ba08daf
```

Comment:
Can we add a note here saying "remove after (insert PR for the updated actions-operator) is merged"?

Reply:
hmm which PR it is

Reply:
found it: # TODO: Unpin this when https://github.com/charmed-kubernetes/actions-operator/pull/46 is merged